card/mod/05_email/set/all/notify.rb

```diff
@@ -68,7 +68,7 @@ def silent_change?
 end
 
 def notable_change?
-  !silent_change? && !supercard && current_act &&
+  !silent_change? && !supercard && current_act && current_act.card_id == id &&
```

Comment:
the suggestion is that the test be something like:

```
act_card? && !silent_change? && ...
```

...and that act_card look like your test:

```
def act_card?
  current_act && current_act.card_id == id
end
```
I'm a little concerned with names here and with how `followable?` works (what if main card isn't followable but subcards are?)

...but I can work on that later.